plugins/uefi-capsule/fu-uefi-grub-device.c

```diff
@@ -0,0 +1,201 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
```

Comment:
2021, and I think I was part of this one too

Reply:
I'm really sorry, I've added you first in that file.